qabelbox/src/main/java/de/qabel/qabelbox/activities/ImageViewerActivity.java

```diff
@@ -50,7 +50,7 @@ private void initViewerFragment() {
         if (viewerFragment == null) {
             Uri uri = getIntent().getParcelableExtra(P_URI);
             String type = getIntent().getStringExtra(P_TYPE);
-            viewerFragment = ImageViewerFragment.newInstance(uri, type);
+            viewerFragment = ImageViewerFragment.Companion.newInstance(uri, type);
```

Comment:
You can put @JVMStatic on the newInstance method, then you don't have to access the Companion explicitly